src/subcommand/server.rs

```diff
@@ -4153,4 +4263,53 @@ mod tests {
       ),
     );
   }
+
+  #[test]
+  fn collections_page_shows_children() {
```

Comment:
I don't think I understand the purpose of this test vs the other test. Could this one be removed?

Reply:
Oh yeah this is extraneous now, doing this in the other one